iaas/20221116.md

```diff
@@ -0,0 +1,74 @@
+# 2022-11-16
+## Participants - use of github handle is preferred
+* @matfechner
+* @master-caster
+* @itrich
+* @ppkuschy
+* @jnull
+* @maliblatt
+* @fkr
+* @ra-beer
+* @maxwolfs
+* @garloff
+* @berendt
+* @o-otte
+* @lossack
+* @tibeer
+
+## Recurring items
+  - Who is presenting in weekly community call and transforming etherpad 
+    to minutes in GitHub? @matfechner
+
+* Anything else to add to the agenda?
+* Fix date for hands-on session (2022-11-25 targeted). Who will take the lead?
```

Comment:
Can you please fix the indentation?

Reply:
Sorry if I was unclear with my comment. I'm referring to the items below "Anything else to add for the agenda?" which should be indented (so that they are nested correctly).